### PR TITLE
#167190444 Implements CRUD functionality on comments

### PR DESCRIPTION
**Pull request by encodedBicoding**
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167190444](https://www.pivotaltracker.com/story/show/167190444)
#### Screenshots (if appropriate)
Screenshot 2019-08-07 at 7 49 38 PM
Screenshot 2019-08-07 at 7 49 45 PM
Screenshot 2019-08-07 at 7 50 02 PM
Screenshot 2019-08-07 at 7 37 28 PM
Screenshot 2019-08-07 at 7 38 30 PM
#### Questions:
N/A

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ config();
 const isProduction = process.env.NODE_ENV === 'production';
 const app = express();
 const log = debug('dev');
-const port = process.env.PORT || 5040;
+const port = process.env.PORT || 5000;
```
Comment (chuxmykel) [inline server/index.js:18]: Is there a reason why this was changed?

**Review by chuxmykel — APPROVED** [review]
Looks good.

**Review by Lundii — APPROVED** [review]
LGTM

File: server/index.js (hunk 1 above)
Comment (encodedBicoding) [reply server/index.js:18]: Yeah, was having too many err:addressinuse

**Review by topseySuave — APPROVED** [review]

**Pull request merged**